decidim-budgets/spec/serializers/project_serializer_spec.rb

```diff
@@ -87,6 +92,30 @@ module Decidim::Budgets
         expect(serialized[:related_proposal_urls]).to include(Decidim::ResourceLocatorPresenter.new(proposals.second).url)
         expect(serialized[:related_proposal_urls]).to include(Decidim::ResourceLocatorPresenter.new(proposals.last).url)
       end
+
+      it "includes the updated date" do
+        expect(serialized).to include(updated_at: project.updated_at)
+      end
+
+      it "includes the follows count" do
+        expect(serialized).to include(follows_count: project.follows_count)
+      end
+
+      it "includes the selected time" do
```

Comment:
To make consistent with "updated date" (line 96)

```suggestion
      it "includes the selected date" do
```

Reply:
whoops!